code/src/OutSystems/Maps/PlacesAPI/SearchPlacesManager.Events.ts

```diff
@@ -7,13 +7,15 @@ namespace OutSystems.Maps.PlacesAPI.SearchPlacesManager.Events {
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
             cb: any;
             event: OSFramework.Event.SearchPlaces.SearchPlacesEventType;
+            uniqueId: string;
```

Comment:
The names should be consistent to avoid confusion between the components unique Id and the even Unique Id

Reply:
I suppose this was left like this because it's already in the event context...

Reply:
Yes, that's why I did simplify the naming. But @gnbm I can be explicitly If it's better.